plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/_core.py

```diff
@@ -130,14 +130,15 @@ def __init__(self, ax_config: AxConfig, max_batch_size: Optional[int]):
 
     def setup(
```

Comment:
We will need to release updates to all the sweepers and launchers when we make the first dev release including this.